webapp/src/data.ts

```diff
@@ -122,7 +122,7 @@ function unsubscribe(component: AnyComponent) {
 
 function expired(ce: CacheEntry) {
     if (!ce.api.expirationTime)
-        return ce.data != null;
+        return !ce.lastRefresh; // needs to be refreshed at least once
```

Comment:
stumbled on this one: if a api is Async with no expirationTime, it is never called.